ic-os/components/setupos-scripts/install-guestos.sh

```diff
@@ -15,12 +15,19 @@ function install_guestos() {
     log_and_halt_installation_on_error "${?}" "Unable to activate HostOS volume group."
 
     TMPDIR=$(mktemp -d)
+    # Release RAM.  Cannot be run concurrently with install-hostos.sh.
+    rm -f disk.img
+    # Extract the disk image to RAM.
+    echo "* Temporarily extracting the GuestOS image to RAM; please stand by for a few seconds"
     tar xafS /data/guest-os.img.tar.zst -C "${TMPDIR}" disk.img
-
-    size=$(wc -c <"${TMPDIR}/disk.img")
-    size="${size:=0}"
-
-    pv -f -s "$size" "${TMPDIR}/disk.img" | dd of=${LV} bs=10M conv=sparse
+    log_and_halt_installation_on_error "${?}" "Unable to extract GuestOS disk-image."
+    # Duplicate the image to the disk.
+    # Progress is handled by status=progress.
+    # Makes a huge difference when running the setup under QEMU with no KVM.
+    # dd will detect nulls in chunks of 4M and sparsify the writes.
+    # In *non-KVM-accelerated* VM, this goes 500 MB/s, three times as fast as before.
+    echo "* Writing the GuestOS image to ${LV}"
+    dd if="${TMPDIR}/disk.img" of=${LV} bs=10M conv=sparse status=progress
```

Comment:
@andrewbattat Do you remember why you chose `pv` over `dd`? I can't get into GitLab to see the discussion on the original MR.

Reply:
https://github.com/dfinity/ic/commit/2f0db1cd004a733ebd920999978ac7fbf4a7de0b
It was for the sake of the installation progress bar (my first real node team task 🥲)

Reply:
The progress bar still exists; we're using `dd` for it now.

I can split those changes to a separate pull request and I will do so.